<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>0</y>
       <width>1920</width>
       <height>1080</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>afficher</string>
      </attribute>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>-340</x>
         <y>-100</y>
         <width>1981</width>
         <height>1091</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">backgrouns:blue;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>110</y>
         <width>681</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(16, 91, 152);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_supprimer">
       <property name="geometry">
        <rect>
         <x>1370</x>
         <y>630</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 font-size: 28 px; /* Sets font size */
    color: black; 
 font: 600 28 px &quot;Sitka Display Semibold&quot;;

        background-color: rgb(16, 91, 152); /* Button background color */
        border: none; /* Remove default border */
        border-radius: 15px; /* Make edges round */
        padding: 10px 20px; /* Add padding */
    }
    QPushButton:hover {
        background-color: #0056b3; /* Darker background on hover */
    }
    QPushButton:pressed {
        background-color: #003f7f; /* Even darker background on press */
    }</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_supprimer">
       <property name="geometry">
        <rect>
         <x>1170</x>
         <y>630</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="stat_bateau">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>630</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 font-size: 28 px; /* Sets font size */
    color: black; 
 font: 600 28 px &quot;Sitka Display Semibold&quot;;

        background-color: rgb(16, 91, 152); /* Button background color */
        border: none; /* Remove default border */
        border-radius: 15px; /* Make edges round */
        padding: 10px 20px; /* Add padding */
    }
    QPushButton:hover {
        background-color: #0056b3; /* Darker background on hover */
    }
    QPushButton:pressed {
        background-color: #003f7f; /* Even darker background on press */
    }</string>
       </property>
       <property name="text">
        <string>stat_bateau</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actionGenerer_Rapport_PDF">
       <property name="geometry">
        <rect>
         <x>1430</x>
         <y>10</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 font-size: 28 px; /* Sets font size */
    color: black; 
 font: 600 28 px &quot;Sitka Display Semibold&quot;;

        background-color: rgb(16, 91, 152); /* Button background color */
        border: none; /* Remove default border */
        border-radius: 15px; /* Make edges round */
        padding: 10px 20px; /* Add padding */
    }
    QPushButton:hover {
        background-color: #0056b3; /* Darker background on hover */
    }
    QPushButton:pressed {
        background-color: #003f7f; /* Even darker background on press */
    }</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_rechercher">
       <property name="geometry">
        <rect>
         <x>1370</x>
         <y>510</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 font-size: 28 px; /* Sets font size */
    color: black; 
 font: 600 28 px &quot;Sitka Display Semibold&quot;;

        background-color: rgb(16, 91, 152); /* Button background color */
        border: none; /* Remove default border */
        border-radius: 15px; /* Make edges round */
        padding: 10px 20px; /* Add padding */
    }
    QPushButton:hover {
        background-color: #0056b3; /* Darker background on hover */
    }
    QPushButton:pressed {
        background-color: #003f7f; /* Even darker background on press */
    }</string>
       </property>
       <property name="text">
        <string>rechercher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_valeurRecherche">
       <property name="geometry">
        <rect>
         <x>1190</x>
         <y>510</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_critere">
       <property name="geometry">
        <rect>
         <x>1090</x>
         <y>510</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>IDC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SERVICE_EFFECTUE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NOM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DESCRIPTION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TYPE_BATEAU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CONTACT_C</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Identifiant : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NOM">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>220</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>360</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Type :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>210</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="type">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>360</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ID">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>150</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>290</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Contact :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="CONTACT">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>290</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_ajouter">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>600</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 font-size: 28 px; /* Sets font size */
    color: black; 
 font: 600 28 px &quot;Sitka Display Semibold&quot;;

        background-color: rgb(16, 91, 152); /* Button background color */
        border: none; /* Remove default border */
        border-radius: 15px; /* Make edges round */
        padding: 10px 20px; /* Add padding */
    }
    QPushButton:hover {
        background-color: #0056b3; /* Darker background on hover */
    }
    QPushButton:pressed {
        background-color: #003f7f; /* Even darker background on press */
    }</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DESCRIPTION">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>430</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_modifier">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>600</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 font-size: 28 px; /* Sets font size */
    color: black; 
 font: 600 28 px &quot;Sitka Display Semibold&quot;;

        background-color: rgb(16, 91, 152); /* Button background color */
        border: none; /* Remove default border */
        border-radius: 15px; /* Make edges round */
        padding: 10px 20px; /* Add padding */
    }
    QPushButton:hover {
        background-color: #0056b3; /* Darker background on hover */
    }
    QPushButton:pressed {
        background-color: #003f7f; /* Even darker background on press */
    }</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>500</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Service effecute :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="SERVICE_EFFECTUE">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>500</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="button_tri">
       <property name="geometry">
        <rect>
         <x>1370</x>
         <y>570</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 font-size: 28 px; /* Sets font size */
    color: black; 
 font: 600 28 px &quot;Sitka Display Semibold&quot;;

        background-color: rgb(16, 91, 152); /* Button background color */
        border: none; /* Remove default border */
        border-radius: 15px; /* Make edges round */
        padding: 10px 20px; /* Add padding */
    }
    QPushButton:hover {
        background-color: #0056b3; /* Darker background on hover */
    }
    QPushButton:pressed {
        background-color: #003f7f; /* Even darker background on press */
    }</string>
       </property>
       <property name="text">
        <string>tri</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_critere_tri">
       <property name="geometry">
        <rect>
         <x>1240</x>
         <y>570</y>
         <width>76</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>IDC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SERVICE_EFFECTUE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TYPE_BATEAU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NOM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CONTACT_C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DESCRIPTION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="Historic">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>550</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 font-size: 28 px; /* Sets font size */
    color: black; 
 font: 600 28 px &quot;Sitka Display Semibold&quot;;

        background-color: rgb(16, 91, 152); /* Button background color */
        border: none; /* Remove default border */
        border-radius: 15px; /* Make edges round */
        padding: 10px 20px; /* Add padding */
    }
    QPushButton:hover {
        background-color: #0056b3; /* Darker background on hover */
    }
    QPushButton:pressed {
        background-color: #003f7f; /* Even darker background on press */
    }</string>
       </property>
       <property name="text">
        <string>historique</string>
       </property>
      </widget>
      <widget class="QLabel" name="historiqueLabel">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>540</y>
         <width>101</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>430</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Description :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>1611</width>
         <height>701</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/pag/pag.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="chatButton">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>570</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>chatButton</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="chatOutput">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>160</y>
         <width>261</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="chatInput">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>380</y>
         <width>241</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <zorder>label_10</zorder>
      <zorder>label_27</zorder>
      <zorder>tableView</zorder>
      <zorder>bt_supprimer</zorder>
      <zorder>lineEdit_supprimer</zorder>
      <zorder>stat_bateau</zorder>
      <zorder>actionGenerer_Rapport_PDF</zorder>
      <zorder>btn_rechercher</zorder>
      <zorder>lineEdit_valeurRecherche</zorder>
      <zorder>combo_critere</zorder>
      <zorder>label_2</zorder>
      <zorder>NOM</zorder>
      <zorder>label_5</zorder>
      <zorder>label_3</zorder>
      <zorder>type</zorder>
      <zorder>ID</zorder>
      <zorder>label_4</zorder>
      <zorder>CONTACT</zorder>
      <zorder>bt_ajouter</zorder>
      <zorder>DESCRIPTION</zorder>
      <zorder>bt_modifier</zorder>
      <zorder>label_6</zorder>
      <zorder>SERVICE_EFFECTUE</zorder>
      <zorder>button_tri</zorder>
      <zorder>combo_critere_tri</zorder>
      <zorder>Historic</zorder>
      <zorder>historiqueLabel</zorder>
      <zorder>label_9</zorder>
      <zorder>chatButton</zorder>
      <zorder>chatOutput</zorder>
      <zorder>chatInput</zorder>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_Confirmer">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>490</y>
         <width>261</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 font-size: 28 px; /* Sets font size */
    color: black; 
 font: 600 28 px &quot;Sitka Display Semibold&quot;;

        background-color: rgb(16, 91, 152); /* Button background color */
        border: none; /* Remove default border */
        border-radius: 15px; /* Make edges round */
        padding: 10px 20px; /* Add padding */
    }
    QPushButton:hover {
        background-color: #0056b3; /* Darker background on hover */
    }
    QPushButton:pressed {
        background-color: #003f7f; /* Even darker background on press */
    }</string>
       </property>
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="mailLogin">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>240</y>
         <width>471</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="PwLogin">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>360</y>
         <width>471</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>1301</width>
         <height>631</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/wav/wav.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>pushButton</zorder>
      <zorder>pushButton_Confirmer</zorder>
      <zorder>mailLogin</zorder>
      <zorder>PwLogin</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
